content-security-policy/connect-src/connect-src-json-import-blocked.sub.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
I'm not sure I understand. Shouldn't this be blocked instead?

Then, as above:

```
<!DOCTYPE html>
<html>

<head>
    <meta http-equiv="Content-Security-Policy" content="connect-src 'none'; script-src 'self' 'unsafe-inline';">
    <script src="/resources/testharness.js"></script>
    <script src="/resources/testharnessreport.js"></script>
</head>

<body>
    <script>
       
        promise_test(t => {
           let spv = new Promise((resolve, reject) => {
            window.addEventListener('securitypolicyviolation', e => {
            if (e.blockedURI === "/// check something in order to be sure that this is the right security policy violation") {
             resolve();
            } else {reject()};
           };
         });

           return Promise.all([
              promise_rejects_js(t, Error,
                import("./resources/dummy.json", { with: { type: "json" } })),
                check_spv
             ]);
          });
    </script>
</body>

</html>
```